operator/tools/tools.go

```diff
@@ -0,0 +1,3 @@
+package tools
+
+// This file only exists to prevent package loading errors for this directory.
```

Comment:
I'm curious why this file necessary if another file (`tools-import.go`) is also present?

Reply:
I think it's because `tools-import.go` are guarded with tag so it could happen that package has no files inside and cause errors.